### PR TITLE
prepare 0.43.0 release

### PR DESCRIPTION
**Pull request by codeboten**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -127,8 +127,8 @@
 - Force usage of `receiverhelper.NewFactory` to implement `component.ReceiverFactory` (#4338)
 - Force usage of `extensionhelper.NewFactory` to implement `component.ExtensionFactory` (#4338)
 - Move `service/parserprovider` package to `config/configmapprovider` (#4206)
-   - Rename `MapProvider` interface to `Provider`
-   - Remove `MapProvider` from helper names
+  - Rename `MapProvider` interface to `Provider`
```
Comment (tigrannajaryan) [inline CHANGELOG.md:130]: Are changes to old version intentional?

**Comment by tigrannajaryan** [comment]
The build failed.

File: CHANGELOG.md (hunk 1 above)
Comment (codeboten) [reply CHANGELOG.md]: No, this was my helpful "format on-save" feature. Reverted.

**Comment by codeboten** [comment]
> The build failed.

@tigrannajaryan fixed, not sure what, if anything, i can do about the codecov failure.

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**